app/views/curation_concerns/base/_media_player.html.erb

```diff
@@ -0,0 +1,51 @@
+<% if @presenter.audio_playlist_presenters.present? %>
```

Comment:
I would maybe put this `if` into the wrapper that calls this instead. 

    <% if @presenter.audio_playlist_presenters.present? %>
      <%= render 'media_player', presenter: @presenter %>
    <% end %>

But I could be wrong or it might not matter and be no better, so I won't hold up the merge any longer!

Reply:
Oh no, but writing that comment alerted me to something else. 

You are passing in a **local** param to `media_presenter` template. That's what `render 'media_player', presenter: @presenter`. Take the iVar `@presenter`, pass it in a local variable `presenter`. In general iVars are hard to keep track of and end up tightly coupling the partial template to the controller -- since iVars are set in the controller. So you gotta know what the controller is doing (which could be many levels up) to know if the iVars are set right. 

So it is considered better practice to pass in a local -- now to understand the context of the partial template, you only you only have to look at where it's called and what locals are passed in.  Use iVars in top-level templates, but locals in partials. (This is one of the messy parts of Rails). 

But to use a local `presenter` passed into a template, it's just in the local variable `presenter`. 

In this template, you're still writing `@presenter`. So you're passing a local argument in... but then not using it, instead using the iVar `@presenter` set by the controller. 

It should be one or the other not both. Probably switch to using local argument `presenter` in this template.